tests/mxnet/test_shared_mem_store.py

```diff
@@ -1,61 +1,93 @@
 import dgl
+import sys
+import random
 import time
 import numpy as np
 from multiprocessing import Process
 from scipy import sparse as spsp
 import mxnet as mx
 import backend as F
 import unittest
+import dgl.function as fn
 
 num_nodes = 100
 num_edges = int(num_nodes * num_nodes * 0.1)
+rand_port = random.randint(5000, 8000)
+print('run graph store with port ' + str(rand_port), file=sys.stderr)
 
-def worker_func(worker_id):
+def check_array_shared_memory(g, worker_id, arrays):
+    if worker_id == 0:
+        for i, arr in enumerate(arrays):
+            arr[0] = i
+        g._sync_barrier()
+    else:
+        g._sync_barrier()
+        for i, arr in enumerate(arrays):
+            assert np.all(arr[0].asnumpy() == i)
+
+def check_init_func(worker_id, graph_name):
     time.sleep(3)
     print("worker starts")
     np.random.seed(0)
     csr = (spsp.random(num_nodes, num_nodes, density=0.1, format='csr') != 0).astype(np.int64)
 
-    g = dgl.contrib.graph_store.create_graph_from_store("test_graph5", "shared_mem")
+    g = dgl.contrib.graph_store.create_graph_from_store(graph_name, "shared_mem", port=rand_port)
```

Comment:
What's the type of `g` here? `GraphStore`?

Reply:
it's a graph store.